src/game_main.cpp

```diff
@@ -229,7 +230,14 @@ int GameMain(const CmdLineSetup_t &setup)
     {
         GameMenu = false;
         LevelSelect = false;
-        FullFileName = setup.testLevel;
+        if(Files::hasSuffix(setup.testLevel, ".rec"))
+        {
+            Record::LoadReplay(setup.testLevel);
+        }
+        else
+        {
+            FullFileName = setup.testLevel;
+        }
```

Comment:
My little tip: Feel free to use braceless condition if not so complex code inside:
```cpp
        if(Files::hasSuffix(setup.testLevel, ".rec"))
            Record::LoadReplay(setup.testLevel);
        else
            FullFileName = setup.testLevel;
```

Also, I highly recommend you to put spaces between condition begin and previous code, also after last brace of condition to make it look more readable:
```cpp
code();

if(cond)
    ko1();
else
    ko2();

code2();
```

Little example that makes harder for quick view and may cause possible misses without proper attention:
```cpp
if(cond)
    ko1();
if(cond)
    ko2();
else if(cond)
    ko2();
if(cond)
    ko2();
else if(cond)
    ko2();
else if(cond)
    ko2();
```

So, better to write as:
```cpp
if(cond)
    ko1();

if(cond)
    ko2();
else if(cond)
    ko2();

if(cond)
    ko2();
else if(cond)
    ko2();
else if(cond)
    ko2();
```

Reply:
P.S. There is a lot of code after Redigit's VB6 which is confusing because of that

Reply:
That is great advice, thank you Wohlstand!